doc/shelley-genesis.md

```diff
@@ -392,127 +495,827 @@ It follows this file layout convention:
+    --cold-signing-key-file example/delegate-keys/delegate1.skey \
+    --operational-certificate-issue-counter example/delegate-keys/delegate-opcert1.counter \
+    --kes-period 0 \
+    --out-file example/node1/cert
+
+$ cardano-cli shelley node issue-op-cert \
+    --hot-kes-verification-key-file example/node2/kes.vkey \
+    --cold-signing-key-file example/delegate-keys/delegate2.skey \
+    --operational-certificate-issue-counter example/delegate-keys/delegate-opcert2.counter \
+    --kes-period 0 \
+    --out-file example/node2/cert
+```
 
 ## Starting a node
 
-TODO
+Now that we have generated our `genesis.json`, KES and VRF operational keys and
+issued ourselves operational certificates, we are close to being able to run
+our nodes.
+
+The last things we need are node configuration and topology files.
+
+For the configuration file, we can start with the default
+[Byron mainnet configuration]
+```
+$ cp cardano-node/configuration/defaults/byron-mainnet/configuration.yaml \
+     example/
+```
+and make a couple tweaks
+```
+$ sed -i 's/Protocol: RealPBFT/Protocol: TPraos/' example/configuration.yaml
+$ sed -i 's/minSeverity: Info/minSeverity: Debug/' example/configuration.yaml
+```
+We will share this `configuration.yaml` file between both nodes we run.
+
+The topology files tell nodes which other nodes to talk to. We will need one
+for each node we run. For this demo we will be running two nodes on the same
+machine, one one port 3001 and the other on 3002. We will configure them to
+talk to each other.
+
+So create the following two files as `example/node1/topology.json` and
+`example/node2/topology.json`.
+```
+{
+  "Producers": [
+    {
+      "addr": "127.0.0.1",
+      "port": 3002,
+      "valency": 1
+    }
+  ]
+}
+```
+```
+{
+  "Producers": [
+    {
+      "addr": "127.0.0.1",
+      "port": 3001,
+      "valency": 1
+    }
+  ]
+}
+```
+So node1 will listen on port 3001 and contact node2 on port 3002, and the other
+way around for node2.
+
+Now we are ready to run our two nodes. With the way we have set them up should
+both be block-producing nodes that take part in the BFT overlay schedule.
+
+One final tweak before we start: it has probably been some time since we
+generated the `genesis.json`, so the start time is probably now some time in
+the past. It is a bit nicer if we set the start time to be shortly in the
+future and then start up our nodes. That way they will all wait for the start
+time and then make blocks from the beginning.
+
+So let's just re-generate our `genesis.json` which will set the start time to
+be 30 seconds into the future, and then we can start our nodes. This assumes
+you did not do any manual tweaking of the generated `genesis.json`, as it will
+be overwritten.
+```
+$ cardano-cli shelley genesis create --genesis-dir example/
+```
+
+So, now in two separate terminal windows we can launch our nodes. Node 1:
+```
+$ cardano-node run \
+    --config example/configuration.yaml \
```

Comment:
maybe it would be more useful to include the configurations to each node's folder in order to have different logs for each node --> `"logs/mainnet.log"` ?

Reply:
True. It would, but then we cannot share the config.